### ISSUE TITLE
Rule Change: [sort-import] disable memberSyntaxSortOrder

### ISSUE THREAD

**Issue by eng-dibo** [eslint/eslint#15957]
### What rule do you want to change?

sort-import

### What change to do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
it should be an option to disable imports by the number of members which in most cases makes no sense but only disturbance.

this plugin has an option to only sort this mess `memberSyntaxSortOrder` but it should also turned off like that `memberSyntaxSortOrder: null`
```

### What does the rule currently do for this code?

only allow sorting `memberSyntaxSortOrder`

### What will the rule do after it's changed?

turn the option `memberSyntaxSortOrder` off

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by mdjermanovic** [comment]
Hi @eng-dibo, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates), so we are not adding new options to the `sort-imports` rule.

You can use the existing option [`"ignoreDeclarationSort": true`](https://eslint.org/docs/rules/sort-imports#ignoredeclarationsort) to disable sorting of declaration statements, and keep sorting of imported members within them.
